### ISSUE TITLE
Warning: Expected server HTML to contain a matching <meta> in <head>

### ISSUE THREAD

**Issue by eran-or** [remix-run/indie-stack#197]
### Have you experienced this bug with the latest version of the template?

yes

### Steps to Reproduce

create a template and then:
1. add to package.json
"dependencies": {
    "@aldabil/react-scheduler": "^2.6.13",
    "@emotion/react": "^11.10.6",
    "@emotion/styled": "^11.10.6",
    "@mui/icons-material": "^5.11.9",
    "@mui/material": "^5.11.10",
    "@mui/x-date-pickers": "^5.0.20",
    "@prisma/client": "^4.9.0",
    "@remix-run/node": "^1.13.0",
    "@remix-run/react": "^1.13.0",
    "@remix-run/serve": "^1.13.0",
    "@remix-run/server-runtime": "^1.13.0",
    "bcryptjs": "^2.4.3",
    "date-fns": "^2.29.3",
    "isbot": "^3.6.5",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "tiny-invariant": "^1.3.1"
  }
2.
create a file somewhere, call it events and add this:
export const EVENTS = [
  {
    event_id: 1,
    title: "Event 1",
    start: new Date(new Date(new Date().setHours(9)).setMinutes(0)),
    end: new Date(new Date(new Date().setHours(10)).setMinutes(0)),
    disabled: true,
    admin_id: [1, 2, 3, 4]
  },
  {
    event_id: 2,
    title: "Event 2",
    start: new Date(new Date(new Date().setHours(10)).setMinutes(0)),
    end: new Date(new Date(new Date().setHours(12)).setMinutes(0)),
    admin_id: 2,
    color: "#50b500"
  },
  {
    event_id: 3,
    title: "Event 3",
    start: new Date(new Date(new Date().setHours(11)).setMinutes(0)),
    end: new Date(new Date(new Date().setHours(12)).setMinutes(0)),
    admin_id: 1,
    editable: false,
    deletable: false
  },
  {
    event_id: 4,
    title: "Event 4",
    start: new Date(
      new Date(new Date(new Date().setHours(9)).setMinutes(30)).setDate(
        new Date().getDate() - 2
      )
    ),
    end: new Date(
      new Date(new Date(new Date().setHours(11)).setMinutes(0)).setDate(
        new Date().getDate() - 2
      )
    ),
    admin_id: 2,
    color: "#900000"
  },
  {
    event_id: 5,
    title: "Event 5",
    start: new Date(
      new Date(new Date(new Date().setHours(10)).setMinutes(30)).setDate(
        new Date().getDate() - 2
      )
    ),
    end: new Date(
      new Date(new Date(new Date().setHours(14)).setMinutes(0)).setDate(
        new Date().getDate() - 2
      )
    ),
    admin_id: 2,
    editable: true
  },
  {
    event_id: 6,
    title: "Event 6",
    start: new Date(
      new Date(new Date(new Date().setHours(10)).setMinutes(30)).setDate(
        new Date().getDate() - 4
      )
    ),
    end: new Date(new Date(new Date().setHours(14)).setMinutes(0)),
    admin_id: 2
  }
];

3.  create a Schedule component:

import React from "react"
import { Scheduler } from "@aldabil/react-scheduler";
import { EVENTS } from "./events";

export default function Schedule (){
  return <Scheduler events={EVENTS} />;

}
4. On routes/index.tsx: 
import Schedule from '~/features/schedule/Schedule'
5. and rewrite this div:
<div className="mx-auto max-w-7xl sm:px-6 lg:px-8">
        <Schedule />
        </div>  
6. open console on the browser.

### Expected Behavior

should run without errors like in the demo of the @aldabil/react-scheduler at https://v96bd.csb.app/
(codesandbox: https://codesandbox.io/s/react-scheduler-demo-standard-v96bd?file=/src/App.js:54-88) 

### Actual Behavior

The ui is broken

**Comment by mdodge-ecgrow** [comment]
I'm getting this error as well trying to follow Kent's tutorial here: https://egghead.io/lessons/remix-fetch-data-in-remix-using-a-basic-loader

**Comment by mdodge-ecgrow** [comment]
Nevermind, found the issue: https://remix.run/docs/en/v1/pages/gotchas#browser-extensions-injecting-code

TLDR: It was one of my browser addons injecting code into the page.  Disabled most of my addons and it was resolved.  Can also run it in incognito to remove the issue.

**Comment by mdodge-ecgrow** [comment]
This looks interesting too: https://github.com/kiliman/remix-hydration-fix

**Comment by machour** [comment]
Closing this as we have several issues about the hydration issues.
